pkg/jx/cmd/install.go

```diff
@@ -2263,6 +2274,168 @@ func (options *InstallOptions) configureBuildPackMode() error {
 	return ebp.Run()
 }
 
+func (options *InstallOptions) configureLongTermStorageBucket() error {
+
+	if options.IsFlagExplicitlySet("long-term-storage") && !options.Flags.LongTermStorage {
+		return nil
+	}
+
+	if !options.BatchMode && !options.Flags.LongTermStorage {
+		surveyOpts := survey.WithStdio(options.In, options.Out, options.Err)
+		confirm := &survey.Confirm{
+			Message: fmt.Sprint("Do you like to enable Long Term Storage? A GCS Bucket will be created"),
+			Default: true,
+		}
+
+		err := survey.AskOne(confirm, &options.Flags.LongTermStorage, nil, surveyOpts)
+		if err != nil {
+			return errors.Wrap(err, "asking to enable Long Term Storage")
+		}
+	}
+
+	var bucketName string
+	if options.Flags.LongTermStorage {
+
+		err := options.ensureInstallValuesAreFilled()
+		if err != nil {
+			return errors.Wrap(err, "filling install values with cluster information")
+		}
+
+		isWriteEnabled, err := gke.IsGCSWriteRoleEnabled(options.installValues[kube.ClusterName],
+			options.installValues[kube.Zone])
+		if err != nil {
+			return errors.Wrap(err, "checking if we the storage write scope is enabled")
+		}
+
+		if !isWriteEnabled {
+			log.Warn("Write access to Google Cloud Storage is not enabled, skipping long term storage")
```

Comment:
Probably, if they can't use long term storage but they didn't see this error when installing, it could be misleading as they will think the logs will be in the bucket. 

So we will just fail fast when this happens.